browsers/duckduckgo.safariextension/js/background.js

```diff
@@ -139,23 +139,32 @@ var onBeforeNavigation = function (e) {
         return
     }
 
-    // avoid redirect loops:
-    // TODO: this won't work in safari yet:
-    //if (thisTab.httpsRedirects[e.url] >= 7) {
```

Comment:
@laurengarcia why were we waiting for 7 times before bailing? Why can't we bail after one failed attempt?

Reply:
Sorry for the captain obvious response but: because many of the ads on the web redirect several times before reaching final destination. The EFF's version of this waits 10 times. I cut it down to 7.

Reply:
In practical terms: if you lower this any more, you're going to break alot of non-tracker third party content on the web.